os/arch/arm/src/artik053/include/artik053_alc5658_i2c.h

```diff
@@ -16,6 +16,8 @@
  *
  ****************************************************************************/
 
+#ifndef __ARCH_ARM_SRC_ARTIK05X_INCLUDE_ARTIK055_ALC5658_I2C_H
```

Comment:
directory structure is arch/arm/src/artik053/include so better to keep as artik053 itself.